admiral/pkg/clusters/types.go

```diff
@@ -151,6 +154,15 @@ func (r *RemoteRegistry) shutdown() {
 	}
 }
 
+func isAnExcludedAsset(assetName string, excludedAssetList []string) bool {
```

Comment:
Can you convert `excludedAssetList` into a map?
```suggestion
func isAnExcludedAsset(assetName string, excludedIdentityList []string) bool {
```